rrmngmnt/service.py

```diff
@@ -199,7 +199,14 @@ def _execute(self, action):
         ]
         executor = self.host.executor()
         rc, _, _ = executor.run_cmd(cmd, io_timeout=self.timeout)
-        return rc == 0
+
+        rc_result = rc == 0
+        if not rc_result:
+            cmd = ['journalctl', '-u', self.name]
```

Comment:
what if journal is not started/does not exist?

Reply:
I would not add it to this repo, rather check the output yourself if the call failed

Reply:
on all systems with systemctl, journalctl command returns something, even for non-existent service and even when journal service (and socket) is stopped